CHANGELOG.md

```diff
@@ -2,6 +2,21 @@
 
 All notable changes to this project will be documented in this file.
 
+## [2.2.28] - 04.05.2022
+### Changed
+#### Analyzer
+* Improved deny-list regex and customizability
+* Added documentation for existing spaCy models
+* Bugfix in analysis explanation scores
+
+#### Image redactor
```

Comment:
Did we bump Image redactor version as well?

Reply:
It's not mandatory since the only thing changed with the Pipfile.lock (not setup.py) but I did it anyway.